src/WebJobs.Script.WebHost/Middleware/FunctionInvocationMiddleware.cs

```diff
@@ -28,6 +30,15 @@ public FunctionInvocationMiddleware(RequestDelegate next)
             _next = next;
         }
 
+        public static bool IsHomepageDisabled
+        {
+            get
+            {
+                return string.Equals(Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebJobsDisableHomepage),
```

Comment:
Would prefer this to be done via IConfiguration approach so we're not locked to environment variables as the only method of setting this.